src/ShapeCrawler/Tables/ITable.cs

```diff
@@ -39,6 +39,11 @@ public interface ITable : IShape
     ///     Merge neighbor cells.
     /// </summary>
     void MergeCells(ICell cell1, ICell cell2);
+
+    /// <summary>
+    ///     Deletes a column at specified index
+    /// </summary>
+    void RemoveColumn(int columnIndex);
```

Comment:
It is ok! But I think later we should change this to `ITable.Columns.RemoveAt(int index)`.